contacts.py

```diff
@@ -216,9 +214,9 @@ def scan_status(self, data, args):
         ret = {}
         if since:
             ret['since'] = since
-            since = int(unix_time(datetime.datetime.strptime(since, "%Y%m%d%H%M")))
+            since = int(unix_time(datetime.datetime.fromisoformat('2007-04-05T14:30Z'.replace("Z", "+00:00"))))
```

Comment:
What is this line - looks like a hard-coded date to me ?

Reply:
woops.